packages/wallet-frontend/src/lib/api/transfers.ts

```diff
@@ -45,23 +45,32 @@ type SendArgs = z.infer<typeof sendSchema>
 type SendError = ErrorResponse<SendArgs | undefined>
 type SendResponse = Promise<SuccessResponse | SendError>
 
-type RequestSuccessResponse = {
+type TransactionResponse = {
```

Comment:
I think we can rename this to `Transaction` only? 🤔
```suggestion
type Transaction = {
```